src/megaapi.cpp

```diff
@@ -503,6 +503,11 @@ int64_t MegaNode::getPublicLinkCreationTime()
     return 0;
 }
 
+string MegaNode::getWritableLinkAuthKey()
```

Comment:
agreed. will change. Note, getPublicLink yields the ownership. but I'll keep it in `MegaNode` for this one, since it does not require creating anything that MegaNodePrivate doesn't have